sonic_sku_create/sonic_sku_create.py

```diff
@@ -0,0 +1,329 @@
+#! /usr/bin/python
```

Comment:
>python [](start = 12, length = 6)

Explicitly specify python2 or python3.

Reply:
And nowadays it is recommended this ways: [https://stackoverflow.com/a/19305076/2514803](https://stackoverflow.com/a/19305076/2514803)

---
In reply to: [290530856](https://github.com/Azure/sonic-utilities/pull/547#discussion_r290530856) [](ancestors = 290530856)

Reply:
changed to #! /usr/bin/env python2